zookeeper-server/src/test/java/org/apache/zookeeper/test/ObserverMasterTest.java

```diff
@@ -416,8 +398,9 @@ private ZooKeeperAdmin createAdmin(int clientPort) throws IOException {
 
     // This test is known to be flaky and fail due to "reconfig already in progress".
     // TODO: Investigate intermittent testDynamicReconfig failures.
-    // @Test
-    public void testDynamicReconfig() throws InterruptedException, IOException, KeeperException {
+    // @ParameterizedTest
```

Comment:
I would prefer to properly ignore the test (`@Disabled`), so it is listed among the ignored tests.

Reply:
Fixed